spec/jobs/generate_hathifile_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require "spec_helper"
 require "zlib"
+require "faraday"
```

Comment:
I'm not familiar with faraday and don't understand why we're using it here. 
I assume a stdlib Net::HTTP is not sufficient?

Reply:
Net::HTTP probably would be fine for this case, but faraday is somewhat less verbose and (generally) the recommendation for an HTTP library to use.